esp/platform/espp.hpp

```diff
@@ -79,6 +79,7 @@ class CEspServer : public CInterface,
     Owned<IPropertyTree> applicationConfig;
     ILogMsgHandler * m_pLogMsgHandler;
     CEspElevatedLoggingFilter *m_elevatedLoggingFilter;
+    CriticalSection elevatedLoggingCs;
```

Comment:
Even though some do not like to prefix members w/ a m_, it seems to be the standard in this file.